lib/genesis/helpers.go

```diff
@@ -189,7 +190,14 @@ func buildRawMap(m map[string]map[string]interface{}) (map[string]string, error)
 		kv.key = append(kv.key, k)
 		buildRawMapInterface(v, kv)
 
-		key, err := formatKey(kv.key)
+		if reflect.DeepEqual([]string{"balances", "balances"}, kv.key) {
```

Comment:
I was wondering why it uses balances:balances here, but `System Account` in `buildBalances`?

Reply:
This was based on an older substrate genesis file, newer genesis files have `palletBalances:balances`.  I'd like to determine a "standard" way of building these, so I'm open changing this.  Let me know.

Reply:
i see, could you update it to the newer version for now? i think as long as we keep it up to date with substrate's genesis it's good